### DISCUSSION TITLE
Logic nodes for force applications and AI development

### DISCUSSION THREAD

**Discussion by BluePrintRandom** [bulletphysics/bullet3#3524] [General]
I would like a flexible api for pyBullet that calls compiled nodes 

users could program using these nodes, or even randomly generate these nodes and test them in a AI gym with goals.....

If one could compare node graphs somehow that worked, one might be able to write equations that write node graphs that are situational\*

**Comment by matpalm** [comment]
can you elaborate on what a "compiled node" is? perhaps a concrete example?
